src/features/controls.ts

```diff
@@ -217,6 +227,54 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
         {type: String, attribute: 'field-of-view', hasChanged: () => true})
     fieldOfView: string = DEFAULT_FIELD_OF_VIEW;
 
+    @style({
+      intrinsics: {
+        basis: [
+          numberNode(-Infinity, 'rad'),
```

Comment:
The basis value is the number that is multiplied for the purposes of scalar values like a percentage. So, if I specify `200%` for the minimum camera theta, it will be the same as `100%` (since both percentages get applied to `-Infinity`).

In this case that might actually be what you want 🤷‍♂

Reply:
Yes, that's what made the most sense to me.